plugins/newsdownloader.koplugin/epubdownloadbackend.lua

```diff
@@ -578,9 +578,8 @@ function EpubDownloadBackend:createEpub(epub_path, html, url, include_images, me
             -- Process can be interrupted at this point between each image download
             -- by tapping while the InfoMessage is displayed
             -- We use the fast_refresh option from image #2 for a quicker download
-            local go_on = UI:info(T(_("%1\n\nRetrieving image %2 / %3 …"), message, inum, nb_images), inum >= 2)
+            local go_on = UI:info((message and message ~= "" and message .. "\n\n" or "") .. T(_("Retrieving image %1 / %2 …"), inum, nb_images), inum >= 2)
```

Comment:
In frontend/ui/wikipedia.lua this line reads:
```lua
local go_on = UI:info(T(_("Retrieving image %1 / %2 …"), inum, nb_images), inum >= 2)
```

All the other changes merely serve to make these lines identical to `wikipedia.lua`.